web/src/refresh-components/avatars/CustomAgentAvatar.tsx

```diff
@@ -18,10 +18,10 @@ interface IconConfig {
   className?: string;
 }
 
-const iconMap: Record<string, IconConfig> = {
-  search: { Icon: SvgSearch, className: "stroke-green-500" },
-  check: { Icon: SvgCheck, className: "stroke-green-500" },
-  code: { Icon: SvgCode, className: "stroke-orange-500" },
+export const iconMap: Record<string, IconConfig> = {
```

Comment:
Can you update this variable name? iconMap isnt too descriptive if we are now exporting this
 ```suggestion
export const avatarIconMap: Record<string, IconConfig> = {
```